web-animations-1/Overview.bs

```diff
@@ -816,6 +816,13 @@ which is used to fix the animation's output <a>time value</a>, called its
 <a>current time</a>, in circumstances such as pausing.
 The <a>hold time</a> is initially <a>unresolved</a>.
 
+In a addition to <a>hold time</a>, an <a>animation</a> maintains a
+<dfn>hold phase</dfn> which is set along with the <a>hold time</a> to determine
+the <a>current phase</a> of an <a>animation</a> with a fixed
+<a>current time</a>. The <a>hold phase</a> has the same range of values as the
+[=timeline phase=] and can be set or unset. The <a>hold phase</a> is initially
+unset.
```

Comment:
Nit for future reference as much as anything, in specs we try to use semantic line breaking where possible. This makes diffs smaller and easier to read.

There are a few articles about this like:
https://rhodesmill.org/brandon/2012/one-sentence-per-line/
https://ramshankar.org/blog/posts/2019/semantic-line-breaks/

We haven't done this very consistently in this spec so far, but going forward we should try to at least start each sentence on a new line (and preferably also after each comma, clause etc.).

Reply:
I will try to remember that going forward. Thanks!